topics/wordplate/index.md

```diff
@@ -4,7 +4,7 @@ created_by: Vincent Klaiber
 display_name: WordPlate
 github_url: https://github.com/wordplate
 logo: wordplate.png
-released: March 18, 2015
+released: October 4, 2013
```

Comment:
If this is the initial commit date, does it make sense for this field? API.md describes `released` as being the date of first release.

Reply:
It depends. March 18, 2015 was the day we created a tag and submitted it to Packagist. Before that it was a boilerplate without tags.

Reply:
Hmm, was it publicly available October 4, 2013 and in a working state? Even if it wasn't on Packagist at that date, if it was at least an alpha that others could use, that seems fine.

Reply:
Yes, it was working fine on October 4, 2013.